src/api/aneel/datastore/search.py

```
@@ -0,0 +1,212 @@
+            dict (dict): O dicionário do qual o valor será extraído.
+            key (str): A chave a ser procurada no dicionário.
+
+        Returns:
+            Any | None: O valor associado à chave, ou None se a chave não for
+                        encontrada ou o valor for avaliado como falso.
+        """
+        __value = dict.get(key)
+        if __value:
+            return __value
+
+        _warnings.warn('Próxima pagina não encontrada.')
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)